save-preprocessor/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+import org.cqfn.save.buildutils.configureSpringBoot
+
+plugins {
+    kotlin("jvm")
+}
+
+configureSpringBoot()
```

Comment:
Please use `withDataJpa=true` instead of manual configuration

Reply:
I don't need jpa (at least now). By default `withSpringDataJpa` is false